assignment/src/routes/appointments.ts

```diff
@@ -0,0 +1,211 @@
+import express, { Request, Response } from 'express';
+import { Appointment, validateAppointment } from '../models/appointment';
+import { Patient } from '../models/patient';
+import mongoose from 'mongoose';
+
+const router = express.Router();
+
+const messages = {
+    added: 'Appointment added successfully',
+    updated: 'Appointment updated successfully',
+    deleted: 'Appointment deleted successfully',
+    notFound: 'Patient not found',
+    noRecord: 'Record not found'
+}
+
+router.get('/', async (_req: Request, res: Response) => {
```

Comment:
https://expressjs.com/en/guide/routing.html

Please read the docs for more details

Reply:
I beleive You may find use of 'dash ( _ )' inappropriate before 'req'. The dash is used as a TypeScript convention to avoid a warning about unused parameters. 

Let me know if I'm missing out something ...